.github/ISSUE_TEMPLATE/report-bug.md

```diff
@@ -0,0 +1,42 @@
+---
+name: "🐞 Report a Bug"
+about: Report a bug encountered while using the Portmaster
+labels: bug
+---
+
+<!--
+Please disclose security related issues privately to support@safing.io.
+-->
+
+**Pre-Submit Checklist**:
+
+- Check known issues:
+  - [Windows](https://docs.safing.io/portmaster/install/windows#known-issues)
+  - [Linux](https://docs.safing.io/portmaster/install/linux#compatibility)
+- Search for similar/existing issues on Github.
+
+**What happened**:
+
+
+
+**What did you expect to happen?**:
+
+
+
+**How did you reproduce it?**:
+
+
+
+**Debug Information**:
+
+<!--
+Paste debug information below:
+- General issue: Click on "Copy Debug Information" on the Settings page.
+- App related issue: Click on "Copy Debug Information" in the dropdown menu of an app in the Monitor view.
+
+⚠ Please check the "Unexpected Logs" and "Network Connections" sections in the preview mode for sensitive or otherwise private information and remove them!
```

Comment:
I'd switch this sentence around:

⚠ Please remove sensitive/private information from the "Unexpected Logs" and "Network Connections" sections. This is easiest to do in the preview mode.

Another thing we should check is whether GitHub sends these logs to their servers before a users has the chance to remove sensitive stuff.